docs/.vuepress/components/TutorialList.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+    <div v-for="(t, index) in tutorials" :key="t.title" class="tutorial-card">
+      <h3 class="tutorial-title">
+        <a :href="`${tutorialsRepo}${t.slug}/TUTORIAL.md`" target="_blank">{{ t.title }}</a
+        ><span> by {{ t.author }}</span>
+      </h3>
+      <div :class="`badge bg-${index}`" v-for="(tag, index) in t.tags" :key="tag">{{ tag }}</div>
+      <p>{{ t.description }}</p>
+      <p class="description"><strong>Difficulty:</strong> {{ t.level }}</p>
+
+      <p>
+        <a :href="`${tutorialsRepo}${t.slug}/TUTORIAL.md`" target="_blank">Tutorial repository</a>
+      </p>
+      <hr />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+const tutorialsRepo = "https://github.com/zkSync-Community-Hub/tutorials/tree/main/tutorials/";
+
+const tutorials = ref([
+  {
+    title: "Create awesome apps with Gelato",
+    description:
+      "Ut sed dui tristique, elementum enim eu, finibus odio. Mauris congue mi vel magna viverra, vel lacinia leo consequat. Morbi eleifend ex in orci ultricies, ut tincidunt leo posuere. ",
+    tags: ["defi", "oracle", "uniswap"],
```

Comment:
I'm thinking it might be helpful to have a set of defined tags in some file or array somewhere, and people can open a PR to add or remove tags

Reply:
we'll allow authors to include up to 4 tags in the PRs. We could define default ones but we'd do that in the tutorials repo